mpt/core/src/main/java/org/apache/james/mpt/api/ImapFeatures.java

```diff
@@ -28,7 +28,10 @@
 public class ImapFeatures {
     
     public enum Feature {
-        NAMESPACE_SUPPORT
```

Comment:
Ok, you mean use the newly introduced Mailbox capabilities for this ?

Seems nice, I will introduce this commit on monday.